packages/components/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { requireNativeComponent } from 'react-native';
+
+export const FRNShimmer = requireNativeComponent('FRNShimmerView')
+
+export class Shimmer extends React.Component<{}> {
```

Comment:
Should this just be a function component? I don't see why it should be a class.

Reply:
This is a WIP, and needs to be written out to follow our guide for building a new component (https://github.com/microsoft/fluentui-react-native/tree/master/packages/components#building-a-component)

Right now, this is very barebones though